### PR TITLE
Some deployment fixes

### PR DESCRIPTION
**Pull request by hategan**

### CODE REVIEW

File: deployment/deploy/DOMAIN_NAME.sdk [hunk 1]
```diff
@@ -0,0 +1 @@
+testing.exaworks.org
```
Comment (mtitov) [inline deployment/deploy/DOMAIN_NAME.sdk:1]: just wondering is this need to be reflected in config files? (`deployment/docker/fs/etc/psij-testing-service` -> `config-*.json`)

File: deployment/deploy/upgrade.sh [hunk 2]
```diff
@@ -59,13 +59,22 @@ getId() {
...
+    fi
     if [ "$ID" != "" ]; then
         # Make sure everything is up to date
         docker exec -it $ID apt-get update
         docker exec -it $ID apt-get upgrade -y
         # Make sure that all files are there if needed
         docker cp ../docker/fs/. $ID:/tmp/fs/
-        docker exec -it $ID bash -c "echo $TYPE.$DOMAIN_NAME > /etc/hostname"
+        if [ -d ../docker/fs.$TYPE ]; then
+            cp ../docker/fs.$TYPE/. $ID:/tmp/fs/
```
Comment (mtitov) [inline deployment/deploy/upgrade.sh:74]: `docker cp ...`?

File: deployment/deploy/DOMAIN_NAME.sdk (hunk 1 above)
Comment (hategan) [reply deployment/deploy/DOMAIN_NAME.sdk:1]: In principle, yes. There is supposed to be a manual "check and edit configuration files" step that should take care of it. We could automate that, but it's not strictly necessary.

File: deployment/deploy/upgrade.sh (hunk 2 above)
Comment (hategan) [reply deployment/deploy/upgrade.sh]: Thank you! This is why we have peer review. Should be fixed now.

File: deployment/deploy/upgrade.sh [hunk 3]
```diff
@@ -59,13 +59,22 @@ getId() {
...
         # Make sure everything is up to date
         docker exec -it $ID apt-get update
         docker exec -it $ID apt-get upgrade -y
         # Make sure that all files are there if needed
         docker cp ../docker/fs/. $ID:/tmp/fs/
-        docker exec -it $ID bash -c "echo $TYPE.$DOMAIN_NAME > /etc/hostname"
+        if [ -d ../docker/fs.$TYPE ]; then
+            docker cp ../docker/fs.$TYPE/. $ID:/tmp/fs/
+        fi
+        run docker cp ../web/. "$ID:/tmp/web/"
```
Comment (mtitov) [inline deployment/deploy/upgrade.sh:76]: oh, one more check -> no `run`, just `docker cp ...`?

File: deployment/deploy/DOMAIN_NAME.sdk (hunk 1 above)
Comment (mtitov) [reply deployment/deploy/DOMAIN_NAME.sdk:1]: I see, ok, thanks!

File: deployment/deploy/upgrade.sh (hunk 3 above)
Comment (hategan) [reply deployment/deploy/upgrade.sh]: Uh, yeah. Actually run is not defined in upgrade.sh. It was only in deploy.sh, so I removed it from the upgrade script.

**Review by mtitov — APPROVED** [review]
LGTM!

**Pull request merged**